src/app/rengular-components-schema.ts

```diff
@@ -83,6 +83,11 @@ const RENGULAR_REGISTRY: {
   'http://rengular.js.org/schema/ChoiceMenu': {
     component: ChoiceMenuComponent,
     inputs: {
+      'http://rengular.js.org/schema/ChoiceMenu#options': (component, data) => {
```

Comment:
Function Style assignment is not needed here. 

    'http://rengular.js.org/schema/ChoiceMenu#gridCols' : gridCols

is all right.

With the definition above, special interface for the `gridCols`, `ActionOptDef` created here, could be removed.

Reply:
Following your advice, ```gridCols``` is undefined.

Reply:
Sorry, quotes are missing: 

'http://rengular.js.org/schema/ChoiceMenu#gridCols' : 'gridCols'

Such practice is used in other components as well:

https://github.com/chigix/rengular/blob/de329617500e112895f7c9de684fe47814677f0f/src/app/rengular-components-schema.ts#L60

Reply:
I get it. Thank you.